mailpoet/assets/js/src/email-editor/engine/components/styles-sidebar/screen-header.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  __experimentalHStack as HStack,
+  __experimentalVStack as VStack,
+  __experimentalSpacer as Spacer,
+  __experimentalHeading as Heading,
+  __experimentalView as View,
+  __experimentalNavigatorToParentButton as NavigatorToParentButton,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { chevronLeft } from '@wordpress/icons';
+
+type Props = {
+  title: string;
+  description?: string;
+  onBack?: () => void;
+};
+
+/**
+ * Component for displaying the screen header and optional description based on site editor component:
+ * https://github.com/WordPress/gutenberg/blob/7fa03fafeb421ab4c3604564211ce6007cc38e84/packages/edit-site/src/components/global-styles/header.js
+ */
+export function ScreenHeader({ title, description, onBack }: Props) {
+  return (
+    <VStack spacing={0}>
+      <View>
+        <Spacer marginBottom={0} paddingX={4} paddingY={3}>
+          <HStack spacing={2}>
+            <NavigatorToParentButton
+              style={{ minWidth: 24, padding: 0 }}
+              icon={chevronLeft}
+              size="small"
+              aria-label={__('Navigate to the previous view')}
+              onClick={onBack}
+            />
+            <Spacer>
+              {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+              {/* @ts-ignore */}
```

Comment:
I've recently learned about `@ts-expect-error`. Here is [an example usage](https://github.com/mailpoet/mailpoet/commit/ca2ddfbbf99605a9e5339caa52a5c59d2e1adbaf).

The difference is that in case the TS error disappears (e.g. after updating packages) we get notified by TS type checker and we remove the comment. SO I would suggest using it instead of `ts-ignore`

Reply:
Nice. Thanks for this suggestion. I will change it.